src/main/kotlin/calculoRetanguloOOP/Retangulo.kt

```diff
@@ -1,6 +1,6 @@
 package calculoRetanguloOOP
 
-class Retangulo(
+open class Retangulo(
```

Comment:
Se lembre que a propriedade `open` é o mesmo que você falar que essa classe pode ser herdada por outras classes.
Mas nesse caso que você só está fazendo testes, e ela não está sendo herdada por ninguém, significa que esse `open` não está sendo usado

```suggestion
class Retangulo(
```

É uma boa pratica a gente evitar de colocar no código coisas que não iremos usar, para evitar bugs surpresas depois
Por exemplo: se adicionarmos uma lib e a configuramos errado, porém por não r estar usando ela, não percebemos o bug. Quando alguém for utiliza-la, vai receber um erro, mas por um bom tempo, ela vai pensar que é o código dele que está causando o erro e não a lib.

Então para evitar essas situações, é bom sempre nos questionarmos "Isso é realmente necessário?", "Precisamos colocar isso nesse PR?" e etc

Reply:
Mas André se você colocou o `open` por algum motivo, pode trazer a explicação aqui que a gente discute se faz sentido ou não, blz?
Sempre pode ter a possibilidade de eu não ter visto um detalhe e na verdade precisava do `open` :smiley: